auth0/src/main/java/com/auth0/android/authentication/storage/CryptoUtil.java

```diff
@@ -145,6 +146,22 @@ public CryptoUtil(@NonNull Context context, @NonNull Storage storage, @NonNull S
         }
     }
 
+    private KeyStore.PrivateKeyEntry getKeyEntryCompat(KeyStore keyStore)
+            throws KeyStoreException, NoSuchAlgorithmException, UnrecoverableEntryException {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
```

Comment:
Please refactor this to look more like the project's code-style. Read the in-line comments I left you:

```java
if (Build.VERSION.SDK_INT < Build.VERSION_CODES.P) {
   return (KeyStore.PrivateKeyEntry) keyStore.getEntry(KEY_ALIAS, null);
}

//Following code is for API 28+
PrivateKey privateKey = (PrivateKey) keyStore.getKey(KEY_ALIAS, null);
if (privateKey == null) {
    //FIXME: See my review comment below
    return (KeyStore.PrivateKeyEntry) keyStore.getEntry(KEY_ALIAS, null); 
}
//I also moved this since it's used only for the "new" call
Certificate certificate = keyStore.getCertificate(KEY_ALIAS);
return new KeyStore.PrivateKeyEntry(privateKey, new Certificate[]{certificate});
```

Reply:
Refactored to match.